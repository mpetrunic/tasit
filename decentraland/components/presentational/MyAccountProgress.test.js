import React from "react";
import { shallow } from "enzyme";
import MyAccountProgress from "./MyAccountProgress";

describe("MyAccountProgress", () => {
  it("renders the component", async () => {
    expect(shallow(<MyAccountProgress progress={0.5} />)).toMatchSnapshot();
  });
});
