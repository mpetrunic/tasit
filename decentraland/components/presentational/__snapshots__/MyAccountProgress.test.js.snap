// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyAccountProgress renders the component 1`] = `
<Component>
  <ProgressBar
    animated={true}
    animationConfig={
      Object {
        "bounciness": 0,
      }
    }
    animationType="spring"
    borderRadius={4}
    borderWidth={0}
    color="#2196f3"
    height={13.34}
    indeterminate={false}
    indeterminateAnimationDuration={1000}
    progress={0.5}
    style={
      Object {
        "marginTop": 133.4,
      }
    }
    unfilledColor="gray"
    useNativeDriver={false}
    width={825.0000000000001}
  />
  <Component
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
      }
    }
  >
    <Component
      style={
        Object {
          "fontWeight": "800",
        }
      }
    >
      50
      % completed
    </Component>
  </Component>
</Component>
`;
